learn/teams/code-repository.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Code Repository
+id: ""
+---
+---
+
+Source code management is a critical part of the project lifecycle, allowing teams to manage their work with a single source of truth. Team includes support for integrating multiple source code repositories with the system so that you have access to source code and control over where your source code resides. It helps in setting up Continuous Integration (CI) and Continuous Delivery (CD) pipelines. You can add multiple code repositories where the particular project's source code will reside.
```

Comment:
Write about what happens when multiple code repos are configured. 
What happens if new code repo added and marked as default. for old projects and new projects.
What is default code repo and what are the limitations of it.
What is the difference between push to external repo and push from studio(many customers confusing on this)
Does every developer need to provide credentilas while pushing.. how is the commit history look like. You can attach some flow here.